lib/query.js

```diff
@@ -290,16 +263,12 @@ Query.prototype.select = function (bins) {
  * @summary Applies a secondary index filter and/or a predicate expression to
  * the query.
  *
- * @description Use a secondary index filter, a predicate expression, or both,
- * to limit the results returned by the query. This method takes either a
- * secondary index filter created using the {@link
- * module:aerospike/filter|filter module}, or a sequence of predicate
- * expressions created using the {@link module:aerospike/predexp|predexp
- * module} as argument. It can also be called multiple times to add both kinds
- * of filters to the same query. (Max. one of each kind.)
+ * @description Use a secondary index filter to limit the results returned by the query. 
+ * This method takes secondary index filter created using the {@link
+ * module:aerospike/filter|filter module} as argument. 
+ * It can also be called multiple times to add filters to the same query.
```

Comment:
This sentence should be removed. Only a single secondary filter predicate is supported.